collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/story/StoryDataClientImpl.java

```diff
@@ -315,7 +315,7 @@ private void processFeatureData(Feature feature, Issue issue, Map<String, IssueF
 		
 		// sStoryPoints
 		IssueField storyPointsField = fields.get(featureSettings.getJiraStoryPointsFieldName());
-		if (storyPointsField != null && storyPointsField.getValue() != null && !TOOLS.sanitizeResponse(storyPointsField.getValue()).isEmpty()) {
+		if (storyPointsField != null && storyPointsField.getValue() != null && !TOOLS.sanitizeResponse(storyPointsField.getValue()).isEmpty() && isNumeric(TOOLS.sanitizeResponse(storyPointsField.getValue()))) {
 			Double value = Double.parseDouble(TOOLS.sanitizeResponse(storyPointsField.getValue()));
```

Comment:
@irvingwa Instead of checking every string for isNumeric you can catch the numberformatexception.